### PR TITLE
Screens for Ecommerce

### PR DESCRIPTION
**Pull request by manojasarsa**
Created Screens for Ecommerce that includes:
- Home Page
- Authentication Pages ( Login, Logout, Signup, ForgotPwd)
- Cart Page
- Product Listing Page
- Wishlist Page
- Profile Page

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,42 +2,25 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="author" content="Manoj Asarsa" />
+    <meta name="keywords" content="easyBuy, easybuyy e-commerce" />
     <meta name="theme-color" content="#000000" />
```
Comment (msdeshmukh009) [inline public/index.html]: You can actually give your theme color here. It will look good.

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+@import 'https://fonts.googleapis.com/css2?family=Lora:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&display=swap';
+@import 'https://fonts.googleapis.com/css2?family=Marck+Script&display=swap';
+@import 'https://zero-ui.netlify.app/docs/main.css';
```
Comment (msdeshmukh009) [inline src/App.css]: Maybe adding these things as html links will be beneficial for [performance](https://www.geeksforgeeks.org/what-is-the-best-way-to-include-css-file-why-use-import/)

File: src/App.js [hunk 3]
```diff
@@ -1,9 +1,37 @@
...
+      </Routes>
       
+      <Link to = "/" ></Link>
+      <Link to = "/productlist" ></Link>
+      <Link to = "/cart" ></Link>
+      <Link to = "/wishlist" ></Link>
+      <Link to = "/login" ></Link>
+      <Link to = "/logout" ></Link>
+      <Link to = "/signup" ></Link>
+      <Link to = "/forgotpwd" ></Link>
```
Comment (msdeshmukh009) [inline src/App.js]: Why are links here is there any reason for that.

File: src/components/Products/Products.jsx [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+    return (
+        <main className="product_container">
+            <div className="product_heading flex flex_justify_start flex_align_center">
+                <h3>Showing All Products</h3>
+                <p>(Showing 10 Products)</p>
+            </div>
+
+            <div className="product_list">
+
+                <div className="card_image flex">
```
Comment (msdeshmukh009) [inline src/components/Products/Products.jsx:13]: I know you are making static pages here but you could have also directly used data from DB to map here.

File: src/components/ShopNow/ShopNow.jsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import "./shopnow.css";
+import { Link } from "react-router-dom";
+
+const ShopNow = () => {
+    return (
+        <main className="main_main flex flex_col flex_justify_center flex_align_start">
+            <h2>CELEBRATE THIS SUMMER</h2>
+            <h2>30% OFF</h2>
+            {/* <a className="btn btn_secondary">  SHOP NOW</a> */}
```
Comment (msdeshmukh009) [inline src/components/ShopNow/ShopNow.jsx]: Remove this comment if not needed.

**Review by msdeshmukh009 — APPROVED** [review]
The UI is looking really nice. Good to merge!!

File: src/App.js (hunk 3 above)
Comment (manojasarsa) [reply src/App.js]: Removed them now.

File: src/App.css (hunk 2 above)
Comment (manojasarsa) [reply src/App.css]: Implemented.

File: public/index.html (hunk 1 above)
Comment (manojasarsa) [reply public/index.html]: That's a good idea. I'll do that.

**Comment by manojasarsa** [comment]
> The UI is looking really nice. Good to merge!!

Thank you, Mahesh for your feedback!

File: src/components/Category/Category.jsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+            <Link className="nav_item flex flex_col flex_align_center" to="/productlist">Women
+                <img src="https://raw.githubusercontent.com/manojasarsa/easybuy-ecom/dev/images/male.jpg" alt="Women" className="img_responsive adjust_img" />
+            </Link>
+
+            <Link className="nav_item flex flex_col flex_align_center" to="/productlist">Kids
+                <img src="https://raw.githubusercontent.com/manojasarsa/easybuy-ecom/dev/images/male.jpg" alt="Kids" className="img_responsive adjust_img" />
+            </Link>
+
+            <Link className="nav_item flex flex_col flex_align_center" to="/productlist">Best Sellers
+                <img src="https://raw.githubusercontent.com/manojasarsa/easybuy-ecom/dev/images/male.jpg" alt="Best Sellers" className="img_responsive adjust_img" />
```
Comment (mimansha-swarup) [inline src/components/Category/Category.jsx:20]: this is being repeated many times can be broken down to  sub components

**Review by mimansha-swarup — APPROVED** [review]
rest  All Looks good to merge

File: src/components/Category/Category.jsx (hunk 6 above)
Comment (preethasri) [reply src/components/Category/Category.jsx:20]: good to merge

**Pull request merged**